x-pack/platform/plugins/shared/streams_app/public/components/data_management/stream_detail_management/classic.tsx

```diff
@@ -30,7 +30,28 @@ export function ClassicStreamDetailManagement({
 }) {
   const {
     path: { key, subtab },
-  } = useStreamsAppParams('/{key}/management/{subtab}');
+  } = useStreamsAppParams('/{key}/{tab}/{subtab}');
+
+  if (!definition.data_stream_exists) {
+    return (
+      <EuiFlexGroup direction="column">
+        <EuiCallOut
+          title={i18n.translate('xpack.streams.unmanagedStreamOverview.missingDatastream.title', {
+            defaultMessage: 'Data stream missing',
+          })}
+          color="danger"
+          iconType="error"
+        >
+          <p>
+            {i18n.translate('xpack.streams.unmanagedStreamOverview.missingDatastream.description', {
+              defaultMessage:
+                'The underlying Elasticsearch data stream for this classic stream is missing. Recreate the data stream to restore the stream by sending data before using the management features.',
+            })}{' '}
```

Comment:
Is this white space an auto-formatting issue or is on purpose?

Reply:
Added by auto-formatting, removed it